cws-installer/src/main/java/jpl/cws/task/CwsInstaller.java

```diff
@@ -1412,6 +1422,34 @@ private static void showInstallationInfo() {
 		print("Database User                 = " + cws_db_username);
 		print("Database Password             = ****** (hidden) ");
 		print("....................................................................................");
+
+		cws_adaptation_use_shared_db = getPreset("adaptation_use_shared_db");
+		cws_adaptation_db_type = getPreset("adaptation_db_type");
+		cws_adaptation_db_host = getPreset("adaptation_db_host");
+		cws_adaptation_db_port = getPreset("adaptation_db_port");
+		cws_adaptation_db_name = getPreset("adaptation_db_name");
+		cws_adaptation_db_username = getPreset("adaptation_db_username");
+		cws_adaptation_db_password = getPreset("adaptation_db_password");
+		cws_adaptation_db_url = "jdbc:" + cws_adaptation_db_type + "://" + cws_adaptation_db_host + ":" + cws_adaptation_db_port + "/" + cws_adaptation_db_name + "?autoReconnect=true";
+
+		if (cws_adaptation_use_shared_db != null) {
+			if (cws_adaptation_use_shared_db.equalsIgnoreCase("y")) {
+
+				if (cws_adaptation_db_type.equals("mariadb")) {
+					cws_adaptation_db_driver = "org.mariadb.jdbc.Driver";
+				} else {
```

Comment:
not sure if we want to make this assumption implicitly that mysql is always the alternate, but probably correct.  Better to be explicit in my opinion to more future code more robust.

Reply:
Set "mysql" condition